components/provisioner/internal/api/resolver.go

```diff
@@ -95,7 +95,7 @@ func (r *Resolver) RuntimeOperationStatus(ctx context.Context, operationID strin
 	return status, err
 }
 
-func (r *Resolver) CleanupRuntimeData(ctx context.Context, id string) (string, error) {
```

Comment:
I would suggest adding some logging here so that we can know when the operation was called for which Runtime.